src/Package/Impl/Help/HelpWindowPane.cs

```diff
@@ -28,5 +37,64 @@ internal class HelpWindowPane : VisualComponentToolWindow<IHelpVisualComponent>
             ToolBarCommandTarget = new CommandTargetToOleShim(null, Component.Controller);
             base.OnCreate();
         }
+
+        public override bool SearchEnabled => true;
+
+        public override void ProvideSearchSettings(IVsUIDataSource pSearchSettings) {
+            var settings = (SearchSettingsDataSource)pSearchSettings;
+            settings.SearchWatermark = Resources.HelpSearchWatermark;
+            settings.SearchTooltip = Resources.HelpSearchTooltip;
+            settings.SearchStartType = VSSEARCHSTARTTYPE.SST_ONDEMAND;
+            base.ProvideSearchSettings(pSearchSettings);
+        }
+
+        public override IVsSearchTask CreateSearch(uint dwCookie, IVsSearchQuery pSearchQuery, IVsSearchCallback pSearchCallback) {
+            return new HelpSearchTask(dwCookie, pSearchQuery, pSearchCallback);
+        }
+
+        private sealed class HelpSearchTask : VsSearchTask {
+            //private static IEnumerable<Lazy<IRHelpSearchTermProvider>> _termProviders;
+            //private readonly List<string> _terms = new List<string>();
+            private readonly IVsSearchCallback _callback;
+            private IRInteractiveWorkflowProvider _workflowProvider;
+
+            public HelpSearchTask(uint dwCookie, IVsSearchQuery pSearchQuery, IVsSearchCallback pSearchCallback)
+                : base(dwCookie, pSearchQuery, pSearchCallback) {
+                _callback = pSearchCallback;
+                _workflowProvider = VsAppShell.Current.ExportProvider.GetExportedValue<IRInteractiveWorkflowProvider>();
+            }
+
+            protected override async void OnStartSearch() {
+                base.OnStartSearch();
+                _callback.ReportComplete(this, 1);
+
+                var searchString = SearchQuery.SearchString;
+                if (!string.IsNullOrWhiteSpace(searchString)) {
+                    var session = _workflowProvider.GetOrCreate().RSession;
+                    using (var inter = await session.BeginInteractionAsync()) {
+                        await inter.RespondAsync("?" + searchString + Environment.NewLine);
```

Comment:
Use `ExecuteAsync` and `rtvs:::show_help`, like `ShowHelpOnCurrentCommand` (so that it doesn't require an active prompt, and falls back to `??` if `?` didn't work).